crates/interfaces/src/blockchain_tree/error.rs

```diff
@@ -25,6 +25,9 @@ pub enum BlockchainTreeError {
     BlockNumberNotFoundInChain { block_number: BlockNumber },
     #[error("Block hash {block_hash} not found in blockchain tree chain")]
     BlockHashNotFoundInChain { block_hash: BlockHash },
+    // Thrown if the block failed to buffer
+    #[error("Block with hash {block_hash} failed to buffer")]
```

Comment:
nit to display full hash instead of `0x12..21`
```suggestion
    #[error("Block with hash {block_hash:?} failed to buffer")]
```